pootle/apps/pootle_translationproject/models.py

```diff
@@ -221,7 +221,7 @@ def checker(self):
             checkerclasses = [import_func(settings.POOTLE_QUALITY_CHECKER)]
         else:
             checkerclasses = [
-                checks.projectcheckers.get(self.project.checkstyle,
+                PROJECT_CHECKERS.get(self.project.checkstyle,
```

Comment:
While this solves some inconsistency in Pootle codebase (setting checkers in Project is done with `PROJECT_CHECKERS` while here the checker is retrieved using https://github.com/translate/translate/blob/master/translate/filters/checks.py#L2277), IMHO Pootle should be using TTK one if the new checker is going to be added there.